simulator/risc_v/t/unit_test.cpp

```diff
@@ -181,6 +181,21 @@ TEST_CASE( "RISCV slo32")
     }
 }
 
+TEST_CASE("RISCV RV32 sloi")
+{
+    CHECK( RISCVInstr<uint32>( 0x20E79793).get_disasm() == "sloi $a5, $a5, 15");
+    std::vector<TestData<uint32>> cases = {
+        TestData<uint32>( 0x1F, 0, 0x1F),
+        TestData<uint32>( 0x17, 2, 0x5F),
+        TestData<uint32>( all_ones<uint32>(), 0xFF, all_ones<uint32<()),
```

Comment:
We've explicitly asked to build and test before submitting any PR.
Why have not you build and test that code?